packages/fxa-settings/src/styles/UnitRow.scss

```diff
@@ -0,0 +1,47 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+.unit-row {
+  @apply flex flex-wrap px-4 pt-7 pb-8;
+
+  &-header {
+    @apply w-full mb-1;
+  }
+
+  &-header-value {
```

Comment:
I'm not sure this needs to indicate that it's the header's value, since it is _the_ value (i.e. there's no main non-header value in this section, is there?). It could even be reduced to `&-content` IMO.

Reply:
Hmm yeah, we can go with `&-content`. I wasn't sure what to call it and since folks are familiar with key/value pairs I went with the "value" of the "header".

Reply:
Ah, actually if we're _just_ talking about the class name it makes sense here - I was going to replace the `headerValue` prop with `content` but that doesn't really work since `{children}` are part of the "content," but I'll make the class name update as it's on the container of `headerValue` and `{children}`.